apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -0,0 +1,354 @@
+import { View, Text, ScrollView, StyleSheet } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import React, { useState, useEffect } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import {
+  Card,
+  IconButton,
+  Divider,
+  Paragraph,
+  TextInput
+} from 'react-native-paper';
+import BigButton from '../components/BigButton';
+import { colors } from '../config/colors';
+
+export const NewQuizScreen = () => {
+  const insets = useSafeAreaInsets();
+  const navigation = useNavigation();
+
+  const [text, setText] = useState('');
+  const [numLines, changeNumLines] = useState(1);
+
+  useEffect(() => {
+    const charLen = text.length;
+    const suggestLines = Math.ceil(charLen / 50);
+    suggestLines > 0 ? changeNumLines(suggestLines) : changeNumLines(1);
+  }, [text]);
+
+  const [numQuestions, setNumQuestions] = useState(1);
+
+  const handleIncrement = () => {
+    setNumQuestions(numQuestions + 1);
+  };
+
+  const handleDecrement = () => {
+    if (numQuestions > 1) {
+      setNumQuestions(numQuestions - 1);
+    }
+  };
+
+  const [selectedQuestionType, setSelectedQuestionType] = useState(null);
+
+  const handleQuestionTypePress = (questionTypeContent) => {
+    setSelectedQuestionType(questionTypeContent);
+  };
+
+  const [selectedDifficulty, setSelectedDifficulty] = useState(null);
+
+  const handleDifficultyPress = (difficultyContent) => {
+    setSelectedDifficulty(difficultyContent);
+  };
+
+  return (
+    <View
+      style={{
+        // Paddings to handle safe area
+        paddingTop: insets.top,
+        paddingBottom: insets.bottom,
+        paddingLeft: insets.left,
+        paddingRight: insets.right
+      }}
+    >
+      <ScrollView>
```

Comment:
My IDE shows this "Create New Quiz" as quite close to the top, perhaps this should be fixed to have more padding/ margin spacing?
![Screenshot_1688873255](https://github.com/new-idea-machine/os-lesson-delivery-system/assets/105380332/a95ee276-f92c-4b30-b778-38f6d7e92d04)

Reply:
Done